CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -19,7 +19,7 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 - Fix [close temp file(s) in case an error happens during defragmentation](https://github.com/etcd-io/etcd/pull/18855)
 
 ### Dependencies
-- Compile binaries using [go 1.22.9](https://github.com/etcd-io/etcd/pull/18850).
+- Compile binaries using [go 1.22.10](https://github.com/etcd-io/etcd/pull/19005).
```

Comment:
Please add a dependencies section in v3.4.36 rather than changing v3.4.35, which has already been released.

Reply:
I have made the changes. Please review them.